nameko/containers.py

```diff
@@ -348,11 +364,15 @@ def _run_worker(self, worker_ctx, handle_result):
 
         with _log_time('ran worker %s', worker_ctx):
 
-            self.dependencies.injections.all.inject(worker_ctx)
+            for dependency in self.dependencies:
```

Comment:
seems a little wasteful to serialise this. let's see if we can think of a nice way to get it back in parallel